x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/editor.tsx

```diff
@@ -283,79 +285,80 @@ export const CustomUrlEditor: FC<CustomUrlEditorProps> = ({
         )}
 
         {(type === URL_TYPE.KIBANA_DASHBOARD || type === URL_TYPE.KIBANA_DISCOVER) &&
-          entityOptions.length > 0 && (
-            <EuiFormRow
-              label={
-                <FormattedMessage
-                  id="xpack.ml.customUrlsEditor.queryEntitiesLabel"
-                  defaultMessage="Query entities"
-                />
-              }
-            >
-              <EuiComboBox
-                placeholder={i18n.translate('xpack.ml.customUrlsEditor.selectEntitiesPlaceholder', {
-                  defaultMessage: 'Select entities',
-                })}
-                options={entityOptions}
-                selectedOptions={selectedEntityOptions}
-                onChange={onQueryEntitiesChange}
-                isClearable={true}
-                data-test-subj="mlJobCustomUrlQueryEntitiesInput"
+        entityOptions.length > 0 ? (
```

Comment:
Just based on something @darnautov brought up before about unexpected things happening because of the nature of `&&` and I thought we all agreed it was better to explicitly return `null`.

Reply:
Removed in 3d93dc50a6a45036a7ff76ee4aed3f9e4a1a8662